documents/src/pages/elements/tree-select.md

```diff
@@ -133,7 +133,7 @@ treeSelect.data = [{
 ```
 ::
 
-The `data` property of the `ef-tree-select` uses the [TreeSelectData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/tree-select/helpers/types.ts) interface for its data items.
+The `data` property of the `ef-tree-select` uses the [TreeSelectData](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/tree-select/helpers/types.ts) interface for its data items. Each item is [TreeDataItem](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/tree/helpers/types.ts) type which is extended from [DataItem](./custom-components/utils/data-management#data-item).
```

Comment:
I think it is `TreeSelectDataItem` instead of `TreeDataItem`

Reply:
Actually, I want to show `TreeDataItem` that the link has more info than `TreeSelectDataItem`. Even if `TreeSelectData` = `TreeDataItem`.

Reply:
+1 on replacing `TreeDataItem ` with `TreeSelectDataItem`. Also, let's follow ComboBox pattern.

Reply:
Updated by replacing `TreeDataItem` with `TreeSelectDataItem`.